project/Mima.scala

```diff
@@ -0,0 +1,10 @@
+import com.typesafe.tools.mima.core._
+import com.typesafe.tools.mima.core.ProblemFilters._
+
+object Mima {
+  val ignoredABIProblems: Seq[ProblemFilter] = {
+    Seq(
+      exclude[MissingClassProblem]("scalafix.package$XtensionRewriteCtx")
```

Comment:
It really shouldn't be there, I kept it as an example of how to filter out errors. I'll remove it.